crates/re_renderer/src/point_cloud_builder.rs

```
@@ -206,22 +212,27 @@ impl<'a> PointCloudBatchBuilder<'a> {
         }
         {
             re_tracing::profile_scope!("colors");
-            let num_written = self.0.color_buffer.extend(colors.take(num_points));
-            // Fill up with defaults. Doing this in a separate step is faster than chaining the iterator.
-            self.0
-                .color_buffer
-                .extend(std::iter::repeat(Color32::TRANSPARENT).take(num_points - num_written));
+            if let Ok(num_written) = self.0.color_buffer.extend(colors.take(num_points)) {
+                // Fill up with defaults. Doing this in a separate step is faster than chaining the iterator.
+                self.0
+                    .color_buffer
+                    .fill_n(Color32::TRANSPARENT, num_points - num_written)
+                    .ok();
+            }
         }
         {
             re_tracing::profile_scope!("picking_instance_ids");
-            let num_written = self
+            if let Ok(num_written) = self
```

This silently swallows an error